src/NCronJob/Scheduler/QueueWorker.cs

```diff
@@ -92,16 +93,16 @@ public override void Dispose()
 
     protected override async Task ExecuteAsync(CancellationToken stoppingToken)
     {
+        AssertUseNCronJobWasCalled();
```

Comment:
This will now throw if `UseNCronJob(Async)` was not called.

@nulltoken / @falvarez1 
Are there any other cases where you know that libraries behave like that?

Reply:
I bagged out for now, as it is a PR on its own and maybe only justified when and if there are more functionality than startup jobs